community/releases/release-manager-runbook/index.md

```diff
@@ -0,0 +1,329 @@
+    [instructions](https://github.com/spinnaker/spin/blob/master/CONTRIBUTING.md#updating-the-gate-api)
+    for updating the generated Gate Client API. Cherry-pick the Gate Client API
+    changes onto the Spin CLI release branch. As of writing, the Swagger
+    Codegen CLI uses 2.3.1; you can get that JAR [here](https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/2.3.1/swagger-codegen-cli-2.3.1.jar).
+    If using a different version, you can try modifying the version parameters
+    in the URL.
+    
+    c. If regenerating the Gate Client API produced any changes, kick off the
+    Flow_BuildAndValidate_1.xx.x for the release branch and wait for a successful
+    completion. This will trigger a downstream Build_PrimaryArtifacts job that
+    we rely on later.
+    
+    d. Run Publish_SpinRelease with the following parameters:
+    
+    - SPIN_BUILD_VERSION_TO_RELEASE: This can be found in the build_spin files
+    written by the Build_PrimaryArtifacts job. Use the version found in the most
+    recent run of the Build_PrimaryArtifacts for the release branch. Note: The
+    major minor part of this version number should match the Gate version for
+    the release branch. If it does not, double check that a tag for the previous
+    minor version of Spin CLI exists. The build auto increments new tags based
+    on the highest pre-existing minor tag.
+    
+    - BOM_VERSION: This is the BOM to associate the Spin CLI release with. It is
+    the latest Spinnaker release number, 1.xx.x.
+
+10. Publish a Sponnet minor version. Run the [publish.sh](https://github.com/spinnaker/spinnaker/blob/master/sponnet/publish.sh)
+script while passing in the version number of the new release.
+Example: VERSION="1.17.2" ./publish.sh
+
+## Every subsequent Monday: Patch a previous Spinnaker version (repeat weeklyish
+for each supported version)
+
+1. Check for any PRs waiting to be [cherry-picked](https://github.com/search?q=org%3Aspinnaker+label%3Acherry-pick+state%3Aopen+type%3Apr).
+
+2. Rerun the Flow_BuildAndValidate_${RELEASE} job and get a green build.
+
+3. Run Publish_SpinnakerPatchRelease:
+
+    a. Enter the major and minor version of the release you’re patching
+    (ex: 1.18) in MAJOR_MINOR_VERSION.
+    b. All other fields can be left as defaults/blank.
+   
+   This looks for a currently active release with this major and minor version.
+   It copies all parameters from that release (name, changelog gist, minimum
+   Halyard version), increments the patch version, and triggers
+   Publish_SpinnakerRelease with these parameters. In general, this is exactly
+   the behavior we want, but if you need to override this behavior (such as to
+   increment the minimum Halyard version in a patch release), you can call
+   Publish_SpinnakerRelease directly and pass the exact parameters that you’d
+   like the new release to have.
+
+4. After the job has completed, run ```hal version list``` and verify that the
```

Comment:
I think just one backtick, for inline. I'm not sure how this would render with the three backticks.